cpp/memilio/io/result_io.h

```diff
@@ -121,6 +121,29 @@ IOResult<void> save_result_with_params(const std::vector<TimeSeries<double>>& re
     return success();
 }
 
+/**
+ * @brief Save the results of the edges for a single graph simulation run.
+ * @param result Simulation results per edge of the graph.
+ * @param ids Identifiers for the start and end node of the edges.
+ * @param filename Name of file
+ * @return Any io errors that occur during writing of the files. 
+ */
+IOResult<void> save_edges(const std::vector<TimeSeries<double>>& results, const std::vector<std::pair<int, int>>& ids,
+                          const std::string& filename);
+
+/**
+ * Save the results for the Edges obtained from the function condense_m_mobility.
+ * @param ensemble_edges Simulation results for each run for each edge.
+ * @param pairs_edges Identifiers for the start and end node of the edges.
+ * @param result_dir Top level directory for all results of the parameter study.
+ * @param save_single_runs [Default: true] Defines if single run results are written.
+ * @param save_single_runs [Default: true] Defines if percentiles are written.
```

Comment:
```suggestion
 * @param save_percentiles [Default: true] Defines if percentiles are written.
```
Just an idea, change this from bool to a default empty vector, so that the user can specify which percentiles are printed?

Reply:
i think this is a good idea. But i will shift this to another issue